lib/article_json/export/apple_news/exporter.rb

```diff
@@ -0,0 +1,31 @@
+module ArticleJSON
+  module Export
+    module AppleNews
+      class Exporter
+        # @param [Array[ArticleJSON::Elements::Base]] elements
+        def initialize(elements)
+          @elements = elements
+        end
+
+        # Return the components section of an Apple News Article as JSON
+        # @return [Hash]
```

Comment:
`to_json` will always return a String...
```suggestion
        # @return [String]
```

Reply:
Amended